homeassistant/components/google_travel_time/helpers.py

```diff
@@ -1,18 +1,46 @@
 """Helpers for Google Time Travel integration."""
+import logging
+
 from googlemaps import Client
 from googlemaps.distance_matrix import distance_matrix
-from googlemaps.exceptions import ApiError
+from googlemaps.exceptions import ApiError, Timeout, TransportError
 
+from homeassistant.core import HomeAssistant
 from homeassistant.helpers.location import find_coordinates
 
+_LOGGER = logging.getLogger(__name__)
+
 
-def is_valid_config_entry(hass, api_key, origin, destination):
+def is_valid_config_entry(
```

Comment:
Nit: Maybe rename now that it's not returning a boolean? `validate_config_entry`?

Reply:
Thanks for catching that!